independent-publisher-connectors/U.K. Government Bank Holidays/apiProperties.json

```diff
@@ -0,0 +1,10 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Martyn Lesbirel",
+    "website": "https://www.gov.uk/bank-holidays",
```

Comment:
@martynlesbirel Please remove `weebiste` property from here as it is not allowed have here.